features/assertions.rb

```diff
@@ -0,0 +1,4 @@
+Then ("I should see {string}") do |content|
+    expect(page).to have_content content
+  end
+  
```

Comment:
this file should be named assertions_steps.rb

Reply:
This is corrected